packages/block-library/src/video/edit.js

```diff
@@ -145,6 +147,12 @@ function VideoEdit( {
 	return (
 		<>
 			<BlockControls>
+				<TracksEditor
+					tracks={ tracks }
+					onChange={ ( newTracks ) => {
+						setAttributes( { tracks: newTracks } );
```

Comment:
Just want to ask whether the shape of the "track" object set here is well defined and that we're not dumping the whole object from the API or something like that.

Reply:
Yes, the shape is very specific an array of objects that may have {src, label, srcLang, kind } properties all being strings. We never dump anything from an API there we are intentional in storing the src explicitly (the other attributes came from user inputs).